templates/python-webapp/app/workflows/generator.py

```
@@ -0,0 +1,113 @@
+                )
+            else:
+                workflow_table.insert(
+                    [
+                        {
+                            "id": f"http-{i}",
+                            "success": False,
+                            "message": f"HTTP failed: {req.status_code}",
+                        }
+                    ]
+                )
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)